forge-gui/res/draft/rankings/rvr.rnk

```diff
@@ -0,0 +1,405 @@
+#1|Liliana, Dreadhorde General|M|RVR
+#2|Aurelia, Exemplar of Justice|M|RVR
+#3|Aurelia, Exemplar of Justice|M|RVR
```

Comment:
Lots of duplicate cards here. There should only be 291 lines. Where did you get the rankings from?

Reply:
It's the exact output of the draft ranking python script in the tools section. The duplicates don't affect the drafting ai. I think it uses the first it finds. My test drafts worked fine.

Reply:
I think the script was updated recently so maybe something needs tweaking there from our end. I'm sure it does work fine but I'd still prefer there be no duplicates

Reply:
I just tested the old script and it gives the same results so it's the page we're pulling from with the issue. If you don't mind I'll push a commit with the correct list